Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp

```diff
@@ -312,4 +355,10 @@ namespace ROS2
         QMessageBox::critical(this, QObject::tr("Error"), errorMessage);
         AZ_Error("RobotImporterWidget", false, "%s", errorMessage.toUtf8().constData());
     }
+
+    bool RobotImporterWidget::IsFileXacro(const AZStd::string& filename) const
+    {
+        return filename.ends_with(".xacro");
```

Comment:
Linux is case sensitive, suggest capitalizing the filename and then checking against an all caps ".XACRO"